tools/pip-requires-client

```diff
@@ -1,16 +1,16 @@
 # All dependencies needed to run rucio client should be defined here
 
 argparse>=1.4.0             # Command-line parsing library
-argcomplete>=1.9.3          # Bash tab completion for argparse
+argcomplete>=1.9.4          # Bash tab completion for argparse
 requests==2.18.4            # Python HTTP for Humans.
 urllib3==1.22               # HTTP library with thread-safe connection pooling, file post, etc.
 dogpile.cache>=0.6.4        # Caching API plugins
 nose>=1.3.7                 # For rucio test-server
 boto>=2.48.0                # S3 boto protocol
-python-swiftclient>=3.4.0   # OpenStack Object Storage API Client Library
-tabulate>=0.8.1             # Pretty-print tabular data
-progressbar2>=3.34.3        # Text progress bar
+python-swiftclient>=3.5.0   # OpenStack Object Storage API Client Library
```

Comment:
```
Downloading/unpacking python-swiftclient>=3.5.0 (from -r tools/pip-requires-client (line 10))
  Downloading python-swiftclient-3.5.0.tar.gz (182kB): 182kB downloaded
  Running setup.py egg_info for package python-swiftclient
    Sorry, Python < 2.7 is not supported for python-swiftclient>=3.0
    Complete output from command python setup.py egg_info:
    Sorry, Python < 2.7 is not supported for python-swiftclient>=3.0

```

Reply:
However, I get the same one with 3.4.0 which we used before?

Reply:
I only get this running with `python-swiftclient<3.0.0`
I find this a bit strange though because I am quite sure that in the tarball we create it had 3.4.0. So I can only assume that technically, the require python 2.7 but that the files are actually still 2.6 compatible.

Reply:
yes, it's just an update but for now we can move them to extra dependencies and drop it. Is the new version fine ?

Reply:
but keep in mind that swift is not used at all for now in the rse protocol classes

Reply:
Yes I think using the EXTRAS mechanism is best, since we are not using swift in protocols anyway.

Reply:
And like this it's clean, since you can also install it (in theory) from a python 2.6 pip.